packages/uhk-web/src/app/components/side-menu/side-menu.component.ts

```diff
@@ -59,6 +62,10 @@ export class SideMenuComponent implements OnInit, OnDestroy {
             this.state = data;
             this.cdRef.markForCheck();
         });
+        this.macroInUseSubscription = this.store.select(getMacroUsage)
```

Comment:
if you manually subscribe to a subscription don't forget to unsubscribe or it will cause a memory leak